specs/vep-2420-getting-started-with-my-data/README.md

```diff
@@ -0,0 +1,248 @@
+
+# VEP-2420: VDK Getting Started with My Data
+
+* **Author(s):** Paul Murphy (paulm2@vmware.com), Antoni Ivanov (aivanov@vmware.com)
+<!-- * **Status:** draft | implementable | implemented | rejected | withdrawn | replaced -->
+* **Status:** draft
+
+
+- [VEP-2420: VDK Getting Started with My Data](#vep-2420-getting-started-with-my-data)
+  - [Summary](#summary)
+  - [Glossary](#glossary)
+  - [Motivation](#motivation)
+    - [Poor assistance around configuration options](#Poor-assistance-around-configuration-options)
+    - [Layers of abstraction aren't clear](#Layers-of-abstraction-are-not-clear)
+    - [Poor documentation around secrets/properties](#Poor-documentation-around-secrets-and-properties)
+    - [Not trivial to run in the IDE](#Not-trivial-to-run-in-the-IDE)
+    - [Reliant on env variables](#Reliant-on-env-variables)
+  - [Requirements and goals](#requirements-and-goals)
+  - [High-level design](#high-level-design)
+  - [API design](#api-design)
+  - [Detailed design](#detailed-design)
+  - [Implementation stories](#implementation-stories)
+  - [Alternatives](#alternatives)
+
+## Summary (Work in progress)
+Users struggle to get started with their own data in VDK.
+When evaluating a data platform like VDK one of the first actions is often to connect it to your own data sources and play around with it.
+Given that this is not a trivial task on VDK it most likely turns a lot of prospective users off.
+The issues range from data jobs being difficult to configure to jobs bring difficult to run.
+On the completion of this initiative user should be able to use VDK to play around with their data in <10 minutes.
+
+## Glossary
+
+## Motivation
+At the moment, VDK users face multiple challenges related to using their own data in VDK.
+Based on user feedback we've received, these can be split into 5 categories.
+
+### Poor assistance around configuration options
+
+VDK documents all config and all config can be seen with 'vdk config-help'.
+However despite this there are a number of issues which make it difficult to configure VDK.
+
+#### Pain points
+Customers often provide the configuration incorrectly.
+1. Either using the wrong names
+2. Putting it in the wrong place
+3. Using values that aren't supported at all
+
+#### Root causes
```

Comment:
Maybe we should mention here the root cause for point 1 of the Pain points (using the wrong names)

Reply:
I would say that the root cause for that is that there is no IDE assistance and That there is no runtime validation. 

These are already listed below. 
Would you agree?